app/models/feedback.rb

```diff
@@ -21,7 +21,7 @@ def self.bike(bike_or_bike_id = nil)
 
   def self.feedback_types
     # Quick semi-hack to get a list of types, good enough till it's not ;)
-    @feedback_types ||= pluck("DISTINCT feedback_type").reject(&:blank?)
+    @feedback_types ||= pluck(Arel.sql("DISTINCT feedback_type")).reject(&:blank?)
```

Comment:
In Rails 5, raw sql emits a warning unless wrapped in `Arel.sql()`

Reply:
I believe there is a pluck distinct method